src/dataset-builder/ConceptSearch.ts

```diff
@@ -70,14 +70,50 @@ export const ConceptSearch = (props: ConceptSearchProps) => {
         style: {
           display: 'flex',
           alignItems: 'center',
-          padding: '2rem 0 0 2rem',
+          padding: '2rem 2rem 0 2rem',
           fontWeight: 800,
           height: '3.5rem',
+          justifyContent: 'space-between',
         },
       },
       [
         searchText.length > 2 ? conceptTableTextForSearch : conceptTableTextForNoSearch,
         ...(conceptsReady ? [] : [h(Spinner, { size: 20, style: { marginLeft: '1rem' } })]),
+        div({ style: { display: 'flex', paddingLeft: 10 } }, [
+          h(
+            PopupTrigger,
+            {
+              side: 'right-aligned-bottom',
+              content: div({ style: { padding: 16, overflowWrap: 'break-word', width: '30em' } }, [
+                strong(['Concept name:']),
+                p({ style: { marginTop: 4, marginBottom: 24 } }, [
```

Comment:
Could you define a constant for these styles as opposed to redefining for each `p` tag?